pytest_django/plugin.py

```diff
@@ -14,17 +14,24 @@
 import py
 import pytest
 
-from .django_compat import is_django_unittest
-from .fixtures import (_django_db_setup, _live_server_helper, admin_client,
-                       admin_user, client, db, django_user_model,
-                       django_username_field, live_server, rf, settings,
-                       transactional_db)
-from .lazy_django import django_settings_is_configured, skip_if_no_django
+from .django_compat import is_django_unittest  # noqa
+from .fixtures import django_db_setup  # noqa
+from .fixtures import django_db_use_migrations  # noqa
+from .fixtures import django_db_keepdb  # noqa
+from .fixtures import django_db_modify_db_settings  # noqa
+from .fixtures import _live_server_helper  # noqa
+from .fixtures import admin_client  # noqa
+from .fixtures import admin_user  # noqa
+from .fixtures import client  # noqa
+from .fixtures import db  # noqa
+from .fixtures import django_user_model  # noqa
+from .fixtures import django_username_field  # noqa
+from .fixtures import live_server  # noqa
+from .fixtures import rf  # noqa
+from .fixtures import settings  # noqa
+from .fixtures import transactional_db  # noqa
```

Comment:
I think this should use the previous style, and then you would only need one `# noqa`, too.